src/services/export.service.ts

```diff
@@ -141,8 +142,12 @@ export class ExportService implements ExportServiceAbstraction {
 
         await Promise.all(promises);
 
+        const testValue = await this.cryptoService.randomNumber(2048, 65536);
+        const encKeyTest = await this.cryptoService.encrypt(testValue.toString());
+
         const jsonDoc: any = {
             encrypted: true,
+            encKeyTest: encKeyTest.encryptedString,
```

Comment:
Just a thought since the import hinges on this MAC.

Perhaps we should call this property `encKeyValidation-DO_NOT_EDIT` since that will result in destroying the export.

Reply:
Yep good point